pkg/alertmanager/api_grafana.go

```diff
@@ -56,7 +57,7 @@ func (gc *UserGrafanaConfig) Validate() error {
 		return errors.New("created must be non-zero")
 	}
 
-	return nil
+	return gc.GrafanaAlertmanagerConfig.AlertmanagerConfig.Validate()
 }
```

Comment:
[ultra-nit-xxl] I preferred the pattern of `if`s followed by `return nil` at the end, but that's stylistic and I won't block on it.